sklearnex/_config.py

```diff
@@ -46,7 +34,7 @@ def get_config():
     set_config : Set global configuration.
     """
     sklearn = skl_get_config()
-    sklearnex = _get_sklearnex_threadlocal_config().copy()
+    sklearnex = _get_onedal_threadlocal_config().copy()
```

Comment:
would calling _get_config() make more sense?

Reply:
good catch. I think it make sense

Reply:
thank you for the proposal! updated onedal's _get_config a bit for it